deploy/sample-backup-configs/sample_config_runtime.ini

```diff
@@ -57,6 +57,10 @@ suffixtag=
 # In general, this is not required to produce deterministic simulations on
 # target machines running linux. Enable if you observe simulation non-determinism.
 zerooutdram=no
+# For designs built with synthesized assertions, enabling (=yes) prints
+# assertion message and terminates simulation if assertion fires.
+# Disabling (=no) ignores assertion firing and continues simulation.
+asserts=yes
```

Comment:
make this more verbose: `capture_synthesized_asserts` or perhaps reversing the polarity would make it more intuitive: `ignore_synthesized_asserts` and matches the variable names in python. 

@timsnyder-siv What name makes sense to you?

Reply:
I think matching the polarity in the python makes sense.  I know some people have a strong dislike of `negative_name=true` semantics.  If you want a more verbose name, something like `asserts_terminate_sim=yes` might make more sense than `asserts=yes`.

`ignore_synthesized_asserts` is okay but I would wonder whether it would still print the message (when I didn't have the nice comment in the sample config).